nl/references/exercise-description/index.md

```diff
@@ -254,13 +254,13 @@ Je kan op Dodona heel eenvoudig codefragmenten tonen in een *monospaced* lettert
 Als je HTML gebruikt, plaats dan `<code>` en `</code>` rond je code.
 
 ```html
-Gebruik de variabele <code>someVariable</code> in je oplossing.
```

Comment:
Why was this changed to snake case? I don't see an obvious reason, so I might be missing something.

Reply:
Python is by far the most popular language on Dodona in terms of exercises and submissions, so I changed it accordingly.